src/ocean/text/json/Json.d

```diff
@@ -890,10 +890,10 @@ class Json(T) : JsonParser!(T)
                 if (val is null)
                     return;
 
+                T[64] tmp = void;
+
                 switch (val.type)
                 {
-                    T[64] tmp = void;
-
```

Comment:
2.066 ? Also this is a (reported) bug. But yeah the workaround is legit.

Reply:
Yep, I checked how things look with 2.066 as 2.077 is not yet available in d-apt

Reply:
Let me ask again: 2.0*6*6 ? With two 6 ? :)

Reply:
:crab: